### PR TITLE
[FEATURE] QA feedback

### PR DESCRIPTION
**Pull request by delete-44**
- [x] Adds "Close options" button

CleanShot 2023-06-02 at 10 44 37

- [x] Update flash messages to prevent page content from jumping

https://github.com/orbit-love/orbit-browser-extension/assets/45462299/b6ac91a3-e2fd-4713-852d-b18a30fcc594

- [x] Disables "save" button if no workspace selected

https://github.com/orbit-love/orbit-browser-extension/assets/45462299/c8296f08-b89a-4cac-9593-21696bc33896

- [x] Widget width standardised for all views so it's less jumpy between loading & rendering

CleanShot 2023-06-02 at 10 50 03

### CODE REVIEW

File: src/options/options.html [hunk 1]
```diff
@@ -84,18 +97,7 @@ <h2 class="text-lg font-semibold text-gray-900">Select a workspace</h2>
...
-              stroke="currentColor" class="w-6 h-6">
-              <path stroke-linecap="round" stroke-linejoin="round"
-                d="M9.813 15.904L9 18.75l-.813-2.846a4.5 4.5 0 00-3.09-3.09L2.25 12l2.846-.813a4.5 4.5 0 003.09-3.09L9 5.25l.813 2.846a4.5 4.5 0 003.09 3.09L15.75 12l-2.846.813a4.5 4.5 0 00-3.09 3.09zM18.259 8.715L18 9.75l-.259-1.035a3.375 3.375 0 00-2.455-2.456L14.25 6l1.036-.259a3.375 3.375 0 002.455-2.456L18 2.25l.259 1.035a3.375 3.375 0 002.456 2.456L21.75 6l-1.035.259a3.375 3.375 0 00-2.456 2.456zM16.894 20.567L16.5 21.75l-.394-1.183a2.25 2.25 0 00-1.423-1.423L13.5 18.75l1.183-.394a2.25 2.25 0 001.423-1.423l.394-1.183.394 1.183a2.25 2.25 0 001.423 1.423l1.183.394-1.183.394a2.25 2.25 0 00-1.423 1.423z" />
-            </svg>
-
-            <figcaption x-text="saveMessage"></figcaption>
-          </figure>
-
-          <fieldset class="mt-5">
+          <fieldset class="mt-5" @change="disableSave = false">
```
Comment (phacks) [inline src/options/options.html:100]: I think we can avoid the extra state variable here. We want to disable the save if the workspace selection is empty, so checking `x-bind:disable="!!selectedWorkspaceSlug"` might be enough to do the trick.

File: src/options/options.html [hunk 2]
```diff
@@ -111,10 +113,10 @@ <h2 class="text-lg font-semibold text-gray-900">Select a workspace</h2>
             </div>
           </fieldset>
 
-          <div class="flex justify-end py-4">
+          <div class="flex flex-row-reverse gap-2 justify-start py-4">
             <button type="button"
-              class="inline-flex gap-x-2 items-center py-2.5 px-3.5 text-sm font-semibold text-white bg-indigo-600 rounded-md shadow-sm hover:bg-indigo-500 focus-visible:outline-2 focus-visible:outline-indigo-600 focus-visible:outline focus-visible:outline-offset-2"
-              @click="save">
+              class="inline-flex gap-x-2 items-center py-2.5 px-3.5 text-sm font-semibold text-white bg-indigo-600 rounded-md hover:bg-indigo-500 focus-visible:outline-2 focus-visible:outline-indigo-600 focus-visible:outline focus-visible:outline-offset-2 disabled:bg-indigo-500"
```
Comment (phacks) [inline src/options/options.html:118]: Would it be possible to make the disabled state colors more explicit? From the video I didn’t notice it was disabled at first.

File: src/options/options.html (hunk 1 above)
Comment (delete-44) [reply src/options/options.html:100]: Afraid not - `selectedWorkspaceSlug` gets set on click of "save". And if the save button is disabled, we can never achieve that.

We could set the selected workspace slug on change of the radio button selection, but then that is saved out of sync with the save button & with everything else, so might cause confusion.

File: src/options/options.html (hunk 2 above)
Comment (delete-44) [reply src/options/options.html:118]: Yep, sure thing!

Comment (delete-44) [reply src/options/options.html]: CleanShot 2023-06-02 at 11 10 20@2x

File: src/options/options.html (hunk 1 above)
Comment (phacks) [reply src/options/options.html:100]: > selectedWorkspaceSlug gets set on click of "save"

I don’t believe so, it appears to be set by Alpine with the `x-model` directive on the radio button inputs. `save` takes that value and adds it to the storage. I gave it a go locally and it seems to work as intended (though it’s `x-bind:disable="!selectedWorkspaceSlug"`

Comment (delete-44) [reply src/options/options.html]: 😱 Oh my gosh. You are 10000% right, thanks for calling that out - I have no *idea* what I was thinking. Too early, clearly. Unneeded state removed, working like a charm with your suggestions

**Review by phacks — APPROVED** [review]

**Pull request merged**